frontend/packages/console-shared/src/utils/multiselectdropdown.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { Select, SelectOption, SelectVariant } from '@patternfly/react-core';
+
+export const MultiSelectDropdown: React.FC<MultiSelectDropdownProps> = ({
+  onChange,
+  placeholder,
+  id,
+  options,
+  defaultSelected,
+}) => {
+  const [isOpen, setIsOpen] = React.useState(false);
+  const [selected, setSelected] = React.useState<string[]>(defaultSelected || []);
+  const onSelect = (event: React.MouseEvent | React.ChangeEvent, selection: string) => {
+    let cSelected: string[] = selected;
+    if (selected.includes(selection)) {
+      cSelected = selected.filter((item) => item !== selection);
+    } else {
+      cSelected = [...selected, selection];
+    }
+    setSelected(cSelected);
+    onChange(cSelected);
+  };
+  const items: JSX.Element[] = options.map((item) => {
+    return <SelectOption key={0} value={item} />;
+  });
+  return (
+    <div>
+      <span id={id} hidden>
```

Comment:
```suggestion
      <span id={id}>
```
Not sure about hidden being used here. If we are trying to hide this element then why render it? if hiding is necessary we should be doing it via CSS files and not through here.

Reply:
done